Dockerfile

```
@@ -30,4 +30,4 @@ ENV LC_ALL=C.UTF-8
 ADD --chown=user:user stack.yaml /home/user/.tmp-haskell/
 ADD --chown=user:user src/main/haskell/kore/package.yaml /home/user/.tmp-haskell/src/main/haskell/kore/
 RUN    cd /home/user/.tmp-haskell \
-    && stack build --only-dependencies --test --bench --haddock --library-profiling
+    && stack build --no-haddock-deps --test --bench --haddock --library-profiling
```

You still need `--only-dependencies` because we are only building the Docker image right now, the source is not checked out.
```suggestion
    && stack build --only-dependencies --no-haddock-deps --test --bench --haddock --library-profiling
```